.changeset/collection.md

```
@@ -0,0 +1,6 @@
+---
+"@daiso-tech/core": minor
+---
+
+Now the instance `validate` method of `ListCollection`, `IterableCollection`, and `AsyncIterableCollection` classes throws `ValidationError`.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)